README.md

```diff
@@ -679,22 +697,45 @@ const serializedStream = new SerializeRowStream()
 
 const firebolt = Firebolt();
 const connection = await firebolt.connect(connectionParams);
-const statement = await connection.execute("select 1 union all select 2");
+const statement = await connection.executeStream("select * from generate_series(1, 1000)");
 
 const { data } = await statement.streamResult();
 
 
 data.pipe(serializedStream).pipe(process.stdout);
 ```
 
-Or use `rowParser` that returns strings or Buffer:
+<a id="in-memory-stream"></a>
+### In-memory stream
+
+When using the streamResult method on the object returned from a simple execute method, the driver will return an in-memory stream of the result. This is useful for small result sets, but not recommended for large result sets.
```

Comment:
Let's add a line explaining what in-memory stream is. Something like "In this case the whole result will be first fetched in memory and then made available via `streamResult`. This is done for compatibility reasons and has no performance benefits compared to using `fetchResult`".
This way we're answering some questions that we've already had from the customers.